lhotse/dataset/collation.py

```diff
@@ -94,18 +192,19 @@ def collate_vectors(
     assert all(len(t.shape) == 1 for t in tensors), "Expected only 1-D input tensors."
     longest = max(tensors, key=lambda t: t.shape[0])
     if matching_shapes:
-        assert all(t.shape == longest.shape for t in tensors), \
-            "All tensors must have the same shape when matching_shapes is set to True."
+        assert all(
+            t.shape == longest.shape for t in tensors
+        ), "All tensors must have the same shape when matching_shapes is set to True."
     result = longest.new_ones(len(tensors), longest.shape[0]) * padding_value
     for i, t in enumerate(tensors):
-        result[i, :t.shape[0]] = t
+        result[i, : t.shape[0]] = t
     return result
 
 
 def collate_matrices(
-        tensors: Iterable[Union[torch.Tensor, np.ndarray]],
-        padding_value: Union[int, float] = 0,
-        matching_shapes: bool = False
+    tensors: Iterable[Union[torch.Tensor, np.ndarray]],
+    padding_value: Union[int, float] = 0,
+    matching_shapes: bool = False,
```

Comment:
which style checker are you using?

Reply:
I used black with default settings.

Reply:
I guess I'll need to make it the chosen autoformatter for Lhotse eventually, as the number of contributors grows. I wonder if it's possible to prevent it from changing unmodified code for now, as that could mess up git blame and makes PRs less readable in terms of actual changes.

Reply:
I will re-write the files anyway, so I ll keep the formatting as it was for this PR in the next push (probably tomorrow).